Fix flaky TestingKafka initialization

It would be better to increase startup attempts or add recreation container logic as Oracle and SQLServer.
```bash
2021-09-24T08:12:14.4760211Z [ERROR] io.trino.plugin.kafka.TestKafkaIntegrationSmokeTest.init  Time elapsed: 32.468 s  <<< FAILURE!
2021-09-24T08:12:14.4762967Z org.testcontainers.containers.ContainerLaunchException: Container startup failed
2021-09-24T08:12:14.4765451Z 	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:334)
2021-09-24T08:12:14.4767740Z 	at org.testcontainers.containers.KafkaContainer.doStart(KafkaContainer.java:97)
2021-09-24T08:12:14.4770217Z 	at org.testcontainers.containers.GenericContainer.start(GenericContainer.java:315)
2021-09-24T08:12:14.4772492Z 	at io.trino.testing.kafka.TestingKafka.start(TestingKafka.java:118)
2021-09-24T08:12:14.4774799Z 	at io.trino.plugin.kafka.KafkaQueryRunnerBuilder.build(KafkaQueryRunnerBuilder.java:68)
2021-09-24T08:12:14.4778122Z 	at io.trino.plugin.kafka.TestKafkaIntegrationSmokeTest.createQueryRunner(TestKafkaIntegrationSmokeTest.java:112)
2021-09-24T08:12:14.4782857Z 	at io.trino.testing.AbstractTestQueryFramework.init(AbstractTestQueryFramework.java:93)
2021-09-24T08:12:14.4785608Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2021-09-24T08:12:14.4787765Z 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2021-09-24T08:12:14.4790966Z 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2021-09-24T08:12:14.4793191Z 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2021-09-24T08:12:14.4795509Z 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
2021-09-24T08:12:14.4797696Z 	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
2021-09-24T08:12:14.4799494Z 	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
2021-09-24T08:12:14.4801164Z 	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
2021-09-24T08:12:14.4803291Z 	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:169)
2021-09-24T08:12:14.4806588Z 	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
2021-09-24T08:12:14.4808505Z 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2021-09-24T08:12:14.4810573Z 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2021-09-24T08:12:14.4812009Z 	at java.base/java.lang.Thread.run(Thread.java:829)
2021-09-24T08:12:14.4813360Z Caused by: org.rnorth.ducttape.RetryCountExceededException: Retry limit hit with exception
2021-09-24T08:12:14.4815818Z 	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:88)
2021-09-24T08:12:14.4818477Z 	at org.testcontainers.containers.GenericContainer.doStart(GenericContainer.java:327)
2021-09-24T08:12:14.4819821Z 	... 19 more
2021-09-24T08:12:14.4821152Z Caused by: org.testcontainers.containers.ContainerLaunchException: Could not create/start container
2021-09-24T08:12:14.4823445Z 	at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:523)
2021-09-24T08:12:14.4826079Z 	at org.testcontainers.containers.GenericContainer.lambda$doStart$0(GenericContainer.java:329)
2021-09-24T08:12:14.4829675Z 	at org.rnorth.ducttape.unreliables.Unreliables.retryUntilSuccess(Unreliables.java:81)
2021-09-24T08:12:14.4831045Z 	... 20 more
2021-09-24T08:12:14.4831846Z Caused by: java.lang.IllegalStateException: Container did not start correctly.
2021-09-24T08:12:14.4833589Z 	at org.testcontainers.containers.GenericContainer.tryStart(GenericContainer.java:461)
2021-09-24T08:12:14.4834890Z 	... 22 more
```

https://github.com/trinodb/trino/runs/3697365158

--- a/testing/trino-testing-kafka/src/main/java/io/trino/testing/kafka/TestingKafka.java
+++ b/testing/trino-testing-kafka/src/main/java/io/trino/testing/kafka/TestingKafka.java
@@ -87,9 +87,11 @@ public final class TestingKafka
         network = Network.newNetwork();
         closer.register(network::close);
         kafka = new KafkaContainer(KAFKA_IMAGE_NAME.withTag(confluentPlatformVersion))
+                .withStartupAttempts(3)
                 .withNetwork(network)
                 .withNetworkAliases("kafka");
         schemaRegistry = new GenericContainer<>(SCHEMA_REGISTRY_IMAGE_NAME.withTag(confluentPlatformVersion))
+                .withStartupAttempts(3)
                 .withNetwork(network)
                 .withNetworkAliases("schema-registry")
                 .withEnv("SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS", "PLAINTEXT://kafka:9092")

